src/interfaces/Config.ts

```diff
@@ -151,7 +151,90 @@ export interface IShapeArgsConfig {
     y?: number;
 }
 
-export interface IDataPoint {
-    dataLabel?: IDataLabelsConfig;
-    shapeArgs?: IShapeArgsConfig;
+export interface IChartOptions {
+    type?: string;
+    stacking?: any;
+    hasStackByAttribute?: boolean;
+    hasViewByAttribute?: boolean;
+    isViewByTwoAttributes?: boolean;
+    legendLayout?: string;
+    xAxes?: any;
+    yAxes?: any;
+    data?: any;
+    actions?: any;
+    grid?: any;
+    xAxisProps?: any;
+    yAxisProps?: any;
+    secondary_xAxisProps?: any;
+    secondary_yAxisProps?: any;
+    title?: any;
+    colorAxis?: Highcharts.ColorAxisOptions;
+    colorAssignments?: IColorAssignment[];
+    colorPalette?: IColorPalette;
+}
+
+export interface IPatternOptionsObject {
+    path: Highcharts.SVGAttributes;
+    width: number;
+    height: number;
+}
+export interface IPatternObject {
+    pattern: IPatternOptionsObject;
+}
+
+export interface IPointData {
```

Comment:
could we extend from IPoint?

Reply:
i combine `IPoint` into `IPointData` and remove `IPoint`.